common/lib/xmodule/xmodule/modulestore/tests/utils.py

```diff
@@ -255,19 +255,19 @@ def build(self, **kwargs):
         """
         contentstore = kwargs.pop('contentstore', None)
         if not contentstore:
-            with self.build_without_contentstore() as (contentstore, modulestore):
+            with self.build_without_contentstore(**kwargs) as (contentstore, modulestore):
```

Comment:
Actually, thinking on item 2, are we avoiding it because of the Django dependency...?